src/blocks/food-and-drinks/test/food-and-drinks.cypress.js

```diff
@@ -99,4 +99,18 @@ describe( 'Test CoBlocks Food and Drinks Block', function() {
 
 		helpers.editPage();
 	} );
+
+	/**
+	 * Test the food-and-drinks block saves heading levels set
+	 */
+	it( 'Updates the inner blocks when the "Heading Level" control is changed.', function() {
+		helpers.addCoBlocksBlockToPage( true, 'food-and-drinks' );
+
+		cy.get( '.block-editor-block-toolbar [aria-label="Change heading level"]' ).click( { force: true } );
+		cy.get( 'div[aria-label="Change heading level"][role="menu"] button' ).contains( 'Heading 2' ).click( { force: true } );
+		cy.get( '[data-type="coblocks/food-and-drinks"] [data-type="coblocks/food-item"] h2' ).should( 'have.length', 2 );
+
+		helpers.savePage();
+		helpers.checkForBlockErrors( 'food-and-drinks' );
+	} );
```

Comment:
We need to assert the default heading level of H4 in order to be confident it has changed to an H2. I've just deployed the change.